msgs_stack/kraken_msgs/action/scan.action

```diff
@@ -0,0 +1,8 @@
+#goal
+float32 speed
```

Comment:
you should also provide which direction to scan +/-1 for clockwise and counter-clockwise